### DISCUSSION TITLE
Where to configure the HS setting ?

### DISCUSSION THREAD

**Discussion by darco415** [Jyers/Marlin#38] [Q&A] [answered]
What setting in configuration.h makes a .bin HS (High Speed)?

**Comment by darco415** [comment]
I think its the xy probe speed which currently shows 10000,,,is that correct?

**Reply by Benetos** [reply]
There are a few. 

Feed rate for one, travel speed between points, multiple probing and stowing the probe or not.

**Reply by darco415** [reply]
ok, I also found bltouch_hs_mode in adv.h,,,,just want to know when I compile my own

**Comment by Jyers** [comment, marked as answer]
For the templates it is just whether or not bltouch_hs_mode is enabled. xy probe speed is something to be tuned at your own discretion.
